### PR TITLE
Add setup instructions

### PR DESCRIPTION
**Pull request by bmaz**
Hello !

I specified the module versions in environment.yml so that everyone has the same installation.
The last downloadable anaconda version offers Python 3.10 so I presumed that everyone would have that version.

### CODE REVIEW

**Comment by mkcor** [comment]
Thanks very much for this, @bmaz!

I'll discuss it with @maikia just to confirm we are on the same page.

**Comment by mkcor** [comment]
Dear @bmaz,

Would you mind resolving the conflicts in `environment.yml` to include pandas (and pin a version for it)?

Thank you!
Marianne

**Comment by bmaz** [comment]
Hello @mkcor ,

Done !

Best,
Béatrice

File: README.md [hunk 1]
```diff
@@ -26,4 +28,26 @@ lessons, which are available under the
...
+1. Open https://docs.conda.io/en/latest/miniconda.html with your web browser.
+2. Click on the latest Installer link depending on your OS: **Miniconda3 Windows 64-bit** for Windows, **Miniconda3 macOS Intel x86 64-bit pkg** for MacOS, **Miniconda3 Linux 64-bit** for Linux.
+3. Install Python 3 by running the Anaconda Installer (double click on the downloaded file) using all of the defaults for installation. Make sure to check **Add Miniconda to my PATH environment variable**.
+4. You can check that Python is installed by opening a terminal and typing `python --version`. The response should be `Python 3.10.x`.
+5. Install the required packages by running the following lines in your terminal:
+```
+conda config --add channels conda-forge
+conda install jupyterlab=3.5.3 matplotlib=3.7.0 ipywidgets=7.6.5 voila=0.3.6 pandas=1.5.3
+```
+
```
Comment (maikia) [inline README.md]: This is great. I would still suggest to add some instructions to check if everything is installed correctly (ie, just like you do with Python but for Jupyter and for importing the libraries within Jupyter).

File: README.md [hunk 2]
```diff
@@ -26,4 +28,26 @@ lessons, which are available under the
 
 ## Setup
 
+### Installing Python Using Anaconda
```
Comment (maikia) [inline README.md]: I would not think that the beginners would be able to judge if to install Anaconda or Miniconda and why they are given two choices. I would suggest just leaving the miniconda instructions

File: README.md (hunk 1 above)
Comment (maikia) [reply README.md]: Maybe also would be worth adding a link to Software Carpentry installation instructions for troubleshooting?

**Review by maikia — COMMENTED** [review]
Thanks @bmaz for this initiative :-)

File: README.md [hunk 3]
```diff
@@ -7,6 +7,8 @@ and uses materials from [Software Carpentry](https://software-carpentry.org)
 lessons, which are available under the
 [CC-BY license](https://creativecommons.org/licenses/by/4.0/).
 
+Please refer to the [Setup](#setup) section to install the required software before the start of the tutorial.
```
Comment (mkcor) [inline README.md]:
```suggestion
Please refer to the [Setup](#setup) section to install the required software ahead of the tutorial.
```

File: README.md (hunk 2 above)
Comment (mkcor) [reply README.md]: I agree with @maikia. I would keep a single set of instructions, namely those using Miniconda.

File: README.md (hunk 1 above)
Comment (mkcor) [reply README.md]: @maikia hmm, the Software Carpentry setup instructions I know of are for [Installing Python Using Anaconda](http://swcarpentry.github.io/python-novice-gapminder/setup.html#installing-python-using-anaconda)... So, pointing to those would contradict our previous review [comment](https://github.com/mkcor/python-prog/pull/7/files#r1156397840).

Instead, I was going to suggest adding a subsection for testing the setup (e.g., see last point of https://datacarpentry.org/image-processing/setup/). The test could consist in launching JupyterLab and running a code cell (I know that students will be learning how to do so in the first episode of this tutorial, but, well) such as:

```py
import pandas as pd                                                            
                                                                               
table = pd.DataFrame(
    {'Time': [0, 1, 2, 3],
     'Emma': [0, 10, 20, 30]}
)  
table.plot();
```
They would copy-paste the above code snippet and check that a plot shows up right below the code cell.

@bmaz does that make sense?

**Comment by mkcor** [comment]
Dear @bmaz,

Are you able to include the proposed changes, say, tonight? Let me know just so I can organize: If not, I'll take over your PR, because I'd like to publish the instructions tomorrow. Thanks!

**Comment by bmaz** [comment]
Hello! I'm writing from my cellphone, so I won't be able to do the changes tonight. Tomorrow before 10:30 is more realistic.
I fully agree about Miniconda and about testing the setup. However, I think that explaining how to open a notebook and how to run a cell is already part of the first lesson. I would go for something simpler, such as `jupyter lab`.

**Comment by bmaz** [comment]
In the end I added the pandas import + plot. 
I also tested the installation on Windows: people need to use the Anaconda prompt and not the standard command line prompt.

**Review by maikia — APPROVED** [review]
LGTM

File: README.md [hunk 4]
```diff
@@ -26,4 +28,26 @@ lessons, which are available under the
 
 ## Setup
 
+1. Open https://docs.conda.io/en/latest/miniconda.html with your web browser.
+2. Click on the latest Installer link depending on your OS: **Miniconda3 Windows 64-bit** for Windows, **Miniconda3 macOS Intel x86 64-bit pkg** for MacOS, **Miniconda3 Linux 64-bit** for Linux.
+3. Install Python 3 by running the Anaconda Installer (double click on the downloaded file) using all of the defaults for installation. On MacOS and Linux, make sure to check **Add Miniconda to my PATH environment variable**.
```
Comment (mkcor) [inline README.md]:
```suggestion
3. Install Python 3 by running the Miniconda Installer (double click on the downloaded file) using all of the defaults for installation. On MacOS and Linux, make sure to check **Add Miniconda to my PATH environment variable**.
```
?

File: README.md [hunk 5]
```diff
@@ -26,4 +28,26 @@ lessons, which are available under the
...
+
```
Comment (mkcor) [inline README.md]:
```suggestion
```
(for consistency)

File: README.md [hunk 6]
```diff
@@ -26,4 +28,26 @@ lessons, which are available under the
 
 ## Setup
 
+1. Open https://docs.conda.io/en/latest/miniconda.html with your web browser.
+2. Click on the latest Installer link depending on your OS: **Miniconda3 Windows 64-bit** for Windows, **Miniconda3 macOS Intel x86 64-bit pkg** for MacOS, **Miniconda3 Linux 64-bit** for Linux.
+3. Install Python 3 by running the Anaconda Installer (double click on the downloaded file) using all of the defaults for installation. On MacOS and Linux, make sure to check **Add Miniconda to my PATH environment variable**.
+   
+4. On Windows: open the **Anaconda Prompt** from the **Start** menu. On MacOs and Linux: open the **Terminal** app. Install the required packages by running the following lines in the opened window:
```
Comment (mkcor) [inline README.md]:
```suggestion
4. On Windows: open the **Anaconda Prompt** from the **Start** menu. On MacOs and Linux: open the **Terminal** app. Install the required packages by running the following lines (copy-paste them in the window that just opened up and press Enter):
```
(being super explicit, as discussed with @maikia)

File: README.md [hunk 7]
```diff
@@ -26,4 +28,26 @@ lessons, which are available under the
...
+```
+conda config --add channels conda-forge
+conda install jupyterlab=3.5.3 matplotlib=3.7.0 ipywidgets=7.6.5 voila=0.3.6 pandas=1.5.3
```
Comment (mkcor) [inline README.md]: No big deal, but could we have the packages in alphabetical order?

File: README.md [hunk 8]
```diff
@@ -26,4 +28,26 @@ lessons, which are available under the
...
+```
+5. Type `jupyter lab` in the Anaconda Prompt / Terminal. After Jupyter Lab has launched, click the “Python 3” button under “Notebook” in the launcher window, or use the “File” menu, to open a new Python 3 notebook.
```
Comment (mkcor) [inline README.md]:
```suggestion
5. Type `jupyter lab` in the Anaconda Prompt / Terminal. After JupyterLab has launched, click the “Python 3” button under “Notebook” in the launcher window, or use the “File” menu, to open a new Python 3 notebook.
```

File: README.md [hunk 9]
```diff
@@ -26,4 +28,26 @@ lessons, which are available under the
...
+6. To test your environment, run the following lines in a cell of the notebook:
```
Comment (mkcor) [inline README.md]:
```suggestion
6. To test your setup, run the following lines in a cell of the notebook:
```

File: README.md [hunk 10]
```diff
@@ -26,4 +28,26 @@ lessons, which are available under the
...
+import pandas as pd                                                            
+                                                                               
+table = pd.DataFrame(
+    {'Time': [0, 1, 2, 3],
+     'Emma': [0, 10, 20, 30]}
+)  
+table.plot();
+```
+
+
```
Comment (mkcor) [inline README.md]:
```suggestion
```

**Review by mkcor — APPROVED** [review]
Amazing! Thank you so much, @bmaz. Just left a few suggestions.

**Pull request merged**